javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/index.html

```diff
@@ -2,124 +2,181 @@
-                x="0px"
-                y="0px"
-              >
-                <g
-                  stroke="none"
-                  stroke-width="1"
+          </div>
+        </div>
+        <div class="player__content">
+          <div id="player__album-art">
+            <img
+              src="https://cdn.freecodecamp.org/MP3%20Player%20Project/Quincy%20Larson%20Album%20Art.jpg"
+              alt="song cover art"
+            />
+          </div>
+          <div class="player__display">
+            <div class="player__display--songArtist">
+              <p id="player__song-title">Cruising for a Musing</p>
+              <p id="player__song-artist">Quincy Larson</p>
+            </div>
+            <div class="player__buttons">
+              <button class="previous">
+                <svg
+                  width="24"
+                  height="14"
+                  viewBox="0 0 24 14"
                   fill="none"
-                  fill-rule="evenodd"
+                  xmlns="http://www.w3.org/2000/svg"
                 >
                   <path
-                    d="M15,18 L15,6 L17,6 L17,18 L15,18 Z M20,18 L20,6 L22,6 L22,18 L20,18 Z M2,5 L13,12 L2,19 L2,5 Z"
-                    fill="#ff0000"
+                    d="M23.9952 0.75L12.7452 7.24519L23.9952 13.7404V0.75Z"
+                    fill="#F5F6F7"
                   />
-                </g>
-              </svg>
-            </button>
-            <button
-              type="button"
-              aria-label="next song"
-              id="next-btn"
-              class="btn-styles"
-            >
-              <svg
-                class="svg-styles"
-                viewBox="0 0 24 24"
-                fill="none"
-                stroke="#ef4444"
-                stroke-width="2"
-                stroke-linecap="round"
-                stroke-linejoin="round"
-              >
-                <polygon points="5 4 15 12 5 20 5 4"></polygon>
-                <line x1="19" y1="5" x2="19" y2="19"></line>
-              </svg>
-            </button>
-            <button
-              type="button"
-              aria-label="shuffle songs"
-              id="shuffle-btn"
-              class="btn-styles"
-            >
-              <svg
-                class="shuffle-btn"
-                xmlns="http://www.w3.org/2000/svg"
-                version="1.0"
-                viewBox="0 0 580.000000 386.000000"
-                preserveAspectRatio="xMidYMid meet"
-              >
-                <g
-                  transform="translate(0.000000,386.000000) scale(0.100000,-0.100000)"
-                  fill="#ff0000"
-                  stroke="none"
+                  <path
+                    d="M11.9952 0.75L0.745186 7.24519L11.9952 13.7404V0.75Z"
+                    fill="#F5F6F7"
+                  />
+                </svg>
+              </button>
+              <button class="play">
+                <img src="assets/play.svg" alt="play" />
```

Comment:
To be more accessible, I think we need to add the currently selected song name and artist to the end of the alt text. 

For the pause button I think it's OK if the alt text is just "pause" as the user should already know which song is playing.